docs/source/tutorial/tutorial-authentication.mdx

```diff
@@ -2,8 +2,6 @@
 title: 7. Enable authentication
 ---
 
-import LoginUISetupPanel from "./components/login_ui_setup_panel.mdx"
-
```

Comment:
This is an import for our documentation javascript, not for Swift. The instructions for setting up the UI will not render without this (See line 111). Please put this back.

Reply:
Will do! Thanks for the heads-up. After going back and looking at some of the other commits, I realized my mistake.